configure.ac

```diff
@@ -250,10 +250,18 @@ DIST_PACKAGING([debian ubuntu],[DEBIAN_PACKAGE],[DEBUILD],[debuild])
 DIST_PACKAGING([osx],[OSX_PACKAGE],[PKGBUILD],[pkgbuild])
 DIST_PACKAGING([bsd],[BSD_PACKAGE],[PKGBUILD],[pkgbuild])
 
+dnl ============================libffi=========================================
+PKG_PROG_PKG_CONFIG()
+PKG_CHECK_MODULES(FFI, [libffi])
+
 dnl Enable pthread library
 AC_CHECK_LIB(pthread, pthread_create,,
     [AC_MSG_ERROR([required library pthread missing])])
 
+dnl Enable dynamic loadable libaries
+AC_CHECK_LIB(dl, dlopen,,
```

Comment:
Is it feasible to make the new dependencies/features optional?

Reply:
Unfortunately not - we need this for the interpreter.